### ISSUE TITLE
Launch error

### ISSUE THREAD

**Issue by elisenlebkuch** [storaged-project/blivet-gui#424]
I don't have any idea what might be causing this, but when I launch the program normally, there opens a tab in the taskbar (or whatever you'd call that on mate), which is named: "blivet-gui is being launched" and has no window belonging to it. This closes after about 15 seconds.
When I run it in the terminal, the ouput is:
```
Traceback (most recent call last):
  File "/usr/bin/blivet-gui", line 38, in <module>
    from blivetgui.dialogs.message_dialogs import ErrorDialog
  File "/usr/lib/python3.11/site-packages/blivetgui/dialogs/message_dialogs.py", line 31, in <module>
    from .helpers import adjust_scrolled_size
  File "/usr/lib/python3.11/site-packages/blivetgui/dialogs/helpers.py", line 34, in <module>
    from blivet import devicefactory
  File "/usr/lib/python3.11/site-packages/blivet/__init__.py", line 30, in <module>
    from . import util, arch
  File "/usr/lib/python3.11/site-packages/blivet/util.py", line 24, in <module>
    gi.require_version("BlockDev", "2.0")
  File "/usr/lib/python3.11/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available for version %s' %
ValueError: Namespace BlockDev not available for version 2.0
```
I use Arch Linux with the Mate desktop.

**Comment by vojtechtrefny** [comment]
You need blivet 3.8.0 or newer. Unfortunately it looks like the latest version available [in AUR is 3.7.1](https://aur.archlinux.org/packages/python-blivet).

blivet-gui can work with older blivet, problem is the new major version of libblockdev (3.0) which is not compatible with older blivet. I am not sure if Arch supports installing older versions of libraries (like for example Debian does) so I am not sure if it would be possible to install the libblockdev 2.x required by blivet 3.7.1.

**Comment by elisenlebkuch** [comment]
You should be able to install older versions of a package, but in this case a huge amount of my other packages has it as a dependency, so I think it would be better to wait for a compatible release of blivet.
